samples/python2/_tutorials_url.txt

```diff
@@ -0,0 +1,2 @@
+OpenCV-Python Tutorials
+http://opencv-python-tutroals.readthedocs.org/en/latest/py_tutorials/py_tutorials.html
```

Comment:
What is about to place this link [here](http://opencv.org/links.html)?

cc @mshabunin